### PR TITLE
Update black in pre-commit hooks, reformat code base, remove python-pandoc

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
- [x] `bumpversion (minor / major / patch)` has been called on this branch
- [ ] Tags have been pushed (`git push --tags`)

* **What kind of change does this PR introduce?** 

This PR makes the xclim code sympatico with recent changes to `black` (v0.19.10 --> v0.20.8). I haven't read the patch notes in-depth (I know there are some interesting experimental features coming, but, in the interest of not messing up everyone's day, they aren't active here yet).

* **Does this PR introduce a breaking change?** 

No. Changes are minor. The convention now appears to be to split `args` and `kwargs` in call signatures vertically by default. You can still write them horizontally, but pre-commit hooks will modify to fit the standard.

* **Other information**:

You may need to perform an update to `pre-commit` if you haven't updated it in a long time. `conda update pre-commit` will do the work for you while the commit hooks will update and manage themselves.

### CODE REVIEW

File: tests/test_run_length.py [hunk 1]
```diff
@@ -79,7 +79,12 @@ def test_start_at_0(self):
 
         # n-dim version versus ufunc
         da3d = xr.open_dataset(self.nc_pr).pr[:, 40:50, 50:68] * 0
-        da3d[0:10,] = da3d[0:10,] + 1
+        da3d[0:10,] = (
+            da3d[
+                0:10,
+            ]
+            + 1
+        )
```
Comment (aulemahal) [inline tests/test_run_length.py:87]: Yo, que pasa? I'm not sure I understand which pep8 rule the line was breaking? It's not the only one-liner that is now spread out for no apparent reason...

**Comment by aulemahal** [comment]
Oh, synchronized commenting. There seems to be something wrong with line lengths?

**Comment by Zeitsperre** [comment]
Ok, yes, some of this is nasty. I'll have time to look deeper into this this afternoon.

**Comment by Zeitsperre** [comment]
Alright, so it's gotten a bit more strict in some cases. If you are slicing lists, you need to remove your trailing commas or else it will explode the statement. Therefore, this:
` da3d[0:10,] = da3d[0:10,] + 1`
Should now be written as this:
`da3d[0:10] = da3d[0:10] + 1`

This appears to be the worst offender. Other changes are slightly jarring, but this is now the standard. Merges should be relatively painless from here.

**Comment by aulemahal** [comment]
Oh! ok so the trailing comma explodes the lists. Is it the same for calls and lists in the other changes?

**Comment by Zeitsperre** [comment]
@aulemahal From what I can tell, if calls signatures have more than two arg/kwargs, they are now exploded/vertical. As I mentioned above, feel free to write it the way you feel most comfortable. If pre-commit is installed correctly, it'll make changes as needed.

**Comment by Zeitsperre** [comment]
Ok, once this clears, feel free to merge!

**Pull request merged**